src/engraving/dom/spanner.cpp

```diff
@@ -1153,6 +1153,45 @@ Measure* Spanner::endMeasure() const
     return toMeasure(m_endElement);
 }
 
+Measure* Spanner::findStartMeasure() const
+{
+    if (!m_startElement) {
+        return nullptr;
+    }
+
+    if (m_startElement->isChordRest()) {
```

Comment:
If I'm not mistaken, I think you could replace this entire method with simply 
`m_startElement->findAncestor(ElementType::MEASURE);`
Same for the end element.